prow/cmd/tide/main.go

```diff
@@ -55,17 +55,27 @@ type options struct {
 	github     prowflagutil.GitHubOptions
 
 	maxRecordsPerPool int
-	// The following are used for reading/writing to GCS.
+	// 	gcsCredentialsFile string is used for reading/writing to GCS block storage.
+	// If you want to write to "file://" paths, this parameter is optional.
+	// If set, this file is used to read/write to gs:// paths
+	// If not, go cloud credential auto-discovery is used
+	// For more details see the pkg/io/v2/providers pkg.
 	gcsCredentialsFile string
+	// 	s3CredentialsFile string is used for reading/writing to s3 block storage.
+	// If you want to write to "file://" paths, this parameter is optional.
+	// If set, this file is used to read/write to s3:// paths
+	// If not, go cloud credential auto-discovery is used
+	// For more details see the pkg/io/v2/providers pkg.
+	s3CredentialsFile string
 	// historyURI where Tide should store its action history.
-	// Can be a /local/path or gs://path/to/object.
+	// Can be file:///local/path, gs://path/to/object or s3://path/to/object.
```

Comment:
I would like to continue supporting `/local/path`

Reply:
see the other conversations